### PR TITLE
Add video to front page and references to side block

### PR DESCRIPTION
**Pull request by maltaesousa**
This PR:
 - Adds a video to front page
 - Replaces "social" block with "references" block
 - Specifies site generation only works on python 3.6

You can see a preview here:
https://sitn.ne.ch/web/gmf/fr/index.html

If you think video is too fast, too slow, doesn't show enough, don't hesitate to give feedback in the PR also.

### CODE REVIEW

**Comment by remyguillaume** [comment]
The video is really great, thanks!

**Review by rbovard — CHANGES_REQUESTED** [review]
Hi @maltaesousa, thanks a lot for the work, it's really nice!

Some remarks about the video:
- I found it a little to fast, is it possible to try a rate between 1 and the one you choose?
- For the features, what do you think to also show query, external WMS, time based layers?
- And a suggestion, maybe add a text (in an overlay or in a blank slide) to name the next feature (full text search, drawing, print, map slider, LiDAR profile)

And about the home page:
- Are you sure it's a good idea to manually convert the page into HTML? I'm affraid that may cause some issues if we update the them or use another one. You had issue with the video and the buttons, correct?
- The title is missing
- The references title is not translated (don't know if it's possible)
- Which criteria do you used to choose the references?

**Comment by maltaesousa** [comment]
>     * I found it a little to fast, is it possible to try a rate between 1 and the one you choose?

I'll make somes tries with slower speed so we can compare

>     * For the features, what do you think to also show query, external WMS, time based layers?

I can do it and I think it's interesting but at PSC some wanted something not too long like ~1minute

>     * And a suggestion, maybe add a text (in an overlay or in a blank slide) to name the next feature (full text search, drawing, print, map slider, LiDAR profile)

The goal is not to do a presentation but just a quick and catchy video. "titles" will make it look boring, no? But it doesn't take too much time to try so i'll make a video with it

>     * Are you sure it's a good idea to manually convert the page into HTML? I'm affraid that may cause some issues if we update the them or use another one. You had issue with the video and the buttons, correct?

Yep, it's impossible to have nice alignements with only markdown. html templates are also supported by pelican, I'm not bypassing pelican build.

>     * The title is missing

It's not mandatory, these html are not "real" html, they're compiled and the title here is translated to an ` h1` that is annoying because it duplicates "Geomapfish"

>     * The references title is not translated (don't know if it's possible)

I'll check if it's possible

>     * Which criteria do you used to choose the references?

The criteria was: "geomapfish in action". Wherever I could find a geomapfish instance running, I put the logo.

**Comment by rbovard** [comment]
> I'll make somes tries with slower speed so we can compare

Great, thanks.

> I can do it and I think it's interesting but at PSC some wanted something not too long like ~1minute

Ha ok, I see. But since it's in infinite loop, it's less important no?

> The goal is not to do a presentation but just a quick and catchy video. "titles" will make it look boring, no? But it doesn't take too much time to try so i'll make a video with it

Okay, forget my suggestion

> Yep, it's impossible to have nice alignements with only markdown. html templates are also supported by pelican, I'm not bypassing pelican build.

My bad, all good thus

> It's not mandatory, these html are not "real" html, they're compiled and the title here is translated to an ` h1` that is annoying because it duplicates "Geomapfish"

I understand, but the generated title is `<title> - GeoMapFish</title>` which is a little weird: image

> I'll check if it's possible

Thanks, but don't loose too many time. A workaround could be to remove the text...

> The criteria was: "geomapfish in action". Wherever I could find a geomapfish instance running, I put the logo. 

And what about cartolacote in this case (poke @pfanguin)? More seriously, why not moving all references links in page `gallery` and add a text like "Take a look on the right block"? In will avoid duplicating links to geoportals, no?

**Comment by maltaesousa** [comment]
@rbovard 

I've made de following changes:
- Translate references title
- Add geocommunes, cartolacote (so sorry @pfanguin !) and aprona to references
- HTML title of page is now nice
- Replace references in gallery by "take a look"

For the video:
- I've put a new version which has a 1.5x speed factor instead of 2x (tell me if you want a different factor, it's easy to change), if you want to compare, old version is still here: https://sitn.ne.ch/web/geomapfish/geomapfish-org/intro_fast.mp4
- If you agree, I'll open a new issue to add query, external WMS, time based layers to the video later cause it takes some time to do it

Changes have been deployed here for review:
https://sitn.ne.ch/web/gmf

File: themes/templates/includes/sidebar/images.html [hunk 1]
```diff
@@ -2,15 +2,17 @@
...
+<div>
   {% if SIDEBAR_IMAGES_HEADER %}
   <h4>{{ title(SIDEBAR_IMAGES_HEADER, DISABLE_SIDEBAR_TITLE_ICONS, icon='external-link-square') }}</h4>
   {% endif %}
-  <ul class="list-group" id="links">
+  <div id="links">
     {% for image in SIDEBAR_IMAGES %}
-    <img width="100%" class="img-thumbnail" src="{{ image }}"/>
+    <div class="mt-1">
+      <a href="{{ image[1] }}" target="_blank"><img width="100%" src="{{ image[0] }}" alt="{{ image[2] }}"/></a>
```
Comment (rbovard) [inline themes/templates/includes/sidebar/images.html]:
```suggestion
      <a href="{{ image[1] }}" target="_blank"><img width="100%" src="{{ image[0] }}" alt="{{ image[2] }} title="{{ image[2] }}"/></a>
```

File: pelicanconf.py [hunk 2]
```diff
@@ -47,17 +49,106 @@
...
+        "WebGIS der Walliser Gemeinden von RUDAZ+PARTNER SA/AG"
+    ],
+    [
+        '/images/references/kantonschwyz.png',
+        'https://map.geo.sz.ch',
+        "WebGIS Kanton Schwyz"
+    ],
+    [
+        '/images/references/geocommunes.png',
+        'http://www.geocommunes.ch/references/',
```
Comment (rbovard) [inline pelicanconf.py]:
```suggestion
        'https://www.geocommunes.ch/references/',
```

**Review by rbovard — COMMENTED** [review]
Thanks a lot for the changes, it's great! I've added two minor code suggestions.

> I've put a new version which has a 1.5x speed factor instead of 2x

I found it better now

> If you agree, I'll open a new issue to add query, external WMS, time based layers to the video later cause it takes some time to do it

Okay

File: content/pages/0_index.html [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+    <div class="row">
+        <div class="col-sm-12">
+            <video autoplay muted class="img-responsive" loop
+                src="https://sitn.ne.ch/web/geomapfish/geomapfish-org/intro.mp4">
+            </video>
+        </div>
+    </div>
+    <div class="row" style="margin-top: 1em;">
+        <div class="col-sm-6">
+            <a href="https://geomapfish-demo-2-7.camptocamp.com" target="_blank" role="button" class="btn btn-primary btn-lg btn-block">Live demo</a>
```
Comment (rbovard) [inline content/pages/0_index.html]:
```suggestion
            <a href="https://geomapfish-demo-2-7.camptocamp.com/?lang=en" target="_blank" role="button" class="btn btn-primary btn-lg btn-block">Live demo</a>
```

File: content/pages/0_index_de.html [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+    <div class="row">
+        <div class="col-sm-12">
+            <video autoplay muted class="img-responsive" loop
+                src="https://sitn.ne.ch/web/geomapfish/geomapfish-org/intro.mp4">
+            </video>
+        </div>
+    </div>
+    <div class="row" style="margin-top: 1em;">
+        <div class="col-sm-6">
+            <a href="https://geomapfish-demo-2-7.camptocamp.com" target="_blank" role="button" class="btn btn-primary btn-lg btn-block">Live demo</a>
```
Comment (rbovard) [inline content/pages/0_index_de.html]:
```suggestion
            <a href="https://geomapfish-demo-2-7.camptocamp.com/?lang=de" target="_blank" role="button" class="btn btn-primary btn-lg btn-block">Live demo</a>
```

File: content/pages/0_index_fr.html [hunk 5]
```diff
@@ -0,0 +1,87 @@
...
+    <div class="row">
+        <div class="col-sm-12">
+            <video autoplay muted class="img-responsive" loop
+                src="https://sitn.ne.ch/web/geomapfish/geomapfish-org/intro.mp4">
+            </video>
+        </div>
+    </div>
+    <div class="row" style="margin-top: 1em;">
+        <div class="col-sm-6">
+            <a href="https://geomapfish-demo-2-7.camptocamp.com" target="_blank" role="button" class="btn btn-primary btn-lg btn-block">Live demo</a>
```
Comment (rbovard) [inline content/pages/0_index_fr.html]:
```suggestion
            <a href="https://geomapfish-demo-2-7.camptocamp.com/?lang=fr" target="_blank" role="button" class="btn btn-primary btn-lg btn-block">Live demo</a>
```

**Review by rbovard — CHANGES_REQUESTED** [review]
And a last code suggestion

File: content/pages/0_index_fr.html [hunk 6]
```diff
@@ -0,0 +1,87 @@
...
+    </p>
+
+    <h2>Caractéristiques</h2>
+
+    <p>
+        Pour une liste plus complétes des caractéristiques, merci de vous référer à la page
+        des <a href="./functionalities">fonctionnalités</a>.
+    </p>
+    <div class="mt-1 flex">
+        <img class="mr-1 no-shrink" alt="webmapping" src="/images/3_2_webmapping.png">
```
Comment (rbovard) [inline content/pages/0_index_fr.html:55]: Is it possible to have the same width (40px) for all this images?

Comment (maltaesousa) [reply content/pages/0_index_fr.html:55]: Yes, but not part of this PR :) I'll open an issue for that

**Comment by maltaesousa** [comment]
@rbovard can I merge? :)

File: content/pages/0_index_fr.html (hunk 6 above)
Comment (rbovard) [reply content/pages/0_index_fr.html:55]: Okay okay ^^

**Review by rbovard — APPROVED** [review]
LGTM, thanks a lot!

**Comment by pfanguin** [comment]
Sorry for my late response. All is good for me ! Thanks a lot !

**Pull request merged**